### PR TITLE
Use sysinfo, add a loader bin

### PR DESCRIPTION
**Pull request by samoylovfp**

### CODE REVIEW

**Comment by samoylovfp** [comment]
Can be manually tested as `cargo run --bin loader | cargo run`

File: src/bin/loader.rs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+use std::time::{Duration, Instant};
```
Comment (demoth) [inline src/bin/loader.rs]: do we need this function? is it the same as `stress`?

File: src/main.rs [hunk 2]
```diff
@@ -1,42 +1,71 @@
...
+        while let Ok(v) = cpu_rx.try_recv() {
+            result.push(v)
+        }
+        result
+    };
+    if !cpu_loads.is_empty() {
+        *last_load = cpu_loads.iter().copied().sum::<f32>() / cpu_loads.len() as f32;
+    }
+    let color = get_color(*last_load / 100.0);
+    let _ignore_error = stdout.set_color(ColorSpec::new().set_fg(Some(color)));
```
Comment (demoth) [inline src/main.rs]: why not `stdout.set_color(ColorSpec::new().set_fg(Some(color)))?;`

File: src/main.rs [hunk 3]
```diff
@@ -1,42 +1,71 @@
...
+) {
+    let cpu_loads = {
+        let mut result = vec![];
+        while let Ok(v) = cpu_rx.try_recv() {
+            result.push(v)
+        }
+        result
+    };
+    if !cpu_loads.is_empty() {
+        *last_load = cpu_loads.iter().copied().sum::<f32>() / cpu_loads.len() as f32;
```
Comment (demoth) [inline src/main.rs:68]: for me lines that have more that 2 expressions are complex enough to have a comment

File: src/main.rs (hunk 2 above)
Comment (demoth) [reply src/main.rs]: or `let _ ...`

File: src/bin/loader.rs (hunk 1 above)
Comment (samoylovfp) [reply src/bin/loader.rs]: Yes but with this we don't need an external binary

Comment (samoylovfp) [reply src/bin/loader.rs]: Replaced with a simple bash script

File: src/main.rs (hunk 2 above)
Comment (samoylovfp) [reply src/main.rs]: This is the same as `let _ = ...`, but it also explains what it means.
Propagating the error is an approach as well, I am not sure what is the right thing to do here
I'll add the question mark

Comment (samoylovfp) [reply src/main.rs]: Done!

File: src/main.rs (hunk 3 above)
Comment (samoylovfp) [reply src/main.rs:68]: Added

File: src/main.rs (hunk 2 above)
Comment (demoth) [reply src/main.rs]: > I am not sure what is the right thing to do here

me neither `¯\_(ツ)_/¯`

**Review by demoth — APPROVED** [review]

**Pull request merged**